src/signals/incident/components/form/MapSelectors/Asset/types.ts

```diff
@@ -83,5 +84,12 @@ export interface DataLayerProps {
   allowClusters?: boolean
 }
 
+export interface AssetSelectRendererProps extends FormFieldProps {
+  meta: Meta
+  handler: any
+  parent: any
+  validatorsOrOpts: any
```

Comment:
These props can be typed with the interfaces from `react-reactive-form`

Reply:
👍

Reply:
doesn't really work unfortunately

Reply:
I'm pretty sure it can typed with
```javascript

import type {
  Handler,
  AbstractControl,
  ValidatorFn,
  AbstractControlOptions,
} from 'react-reactive-form'

...

export interface AssetSelectRendererProps extends FormFieldProps {
  meta: Meta
  handler: Handler
  parent: AbstractControl
  validatorsOrOpts: ValidatorFn | ValidatorFn[] | AbstractControlOptions | null
}
```

Reply:
This is not the case.. type Handler and type AbstractControl are giving problems.